$MirOS: ports/emulators/qemu/patches/patch-hw_sb16_c,v 1.1 2005/05/03 19:47:14 tg Exp $
--- hw/sb16.c.orig	Tue May  3 19:37:28 2005
+++ hw/sb16.c	Tue May  3 19:38:01 2005
@@ -535,8 +535,9 @@ static void command (SB16State *s, uint8
     }
 
     s->cmd = cmd;
-    if (!s->needed_bytes)
+    if (!s->needed_bytes) {
         ldebug ("\n");
+    }
     return;
 
  warn:
@@ -882,8 +883,9 @@ static IO_READ_PROTO (dsp_read)
         goto error;
     }
 
-    if (!ack)
+    if (!ack) {
         ldebug ("read %#x -> %#x\n", nport, retval);
+    }
 
     return retval;
 
@@ -991,8 +993,10 @@ static IO_READ_PROTO(mixer_read)
 #ifndef DEBUG_SB16_MOST
     if (s->mixer_nreg != 0x82)
 #endif
+    {
     ldebug ("mixer_read[%#x] -> %#x\n",
             s->mixer_nreg, s->mixer_regs[s->mixer_nreg]);
+    }
     return s->mixer_regs[s->mixer_nreg];
 }
 
